.classpath

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
I can't remember, did we say we would start/stop adding .classpaths into version control?

Reply:
I think we concluded that it's irrelevant but It's probably best practice to avoid doing that.